netlify.toml

```diff
@@ -8,6 +8,5 @@
   command = "yarn docs:build && yarn docs:export"
 
 [build.environment]
-  YARN_VERSION = "1.21.1"
```

Comment:
This might not be needed anymore. 1.17 is the default in netlify and should already support `yarn-path`.

Reply:
Apparently it's still 1.3:

>  Successfully installed Yarn 1.3.2! Please open another terminal where the `yarn` command will now be available.

-- https://app.netlify.com/sites/material-ui/deploys/5f0ad53f04c0720008e769c8

But even that version supports `yarn-path`.

Seems like 1.17 is only the default for new pages. [Once a page is created the default yarn version is locked](https://community.netlify.com/t/default-yarn-version-is-now-1-17/2297/5?u=eps1lon).